### ISSUE TITLE
Missing Item after completing Relic of the Earthen Ring quest

### ISSUE THREAD

**Issue by dedraik** [dalaranwow/dalaran-wow#4931]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After completing the quest, Relic of the Earthen Ring (you retrieve the quest in Orgrimmar after hitting level 30 and having collected all of the totems), you are suppose to receive a relic that consumes the four basic totems you receive as a shaman.  It allows you to reduce space by consuming the four totems (earth, air, fire, water), is a relic piece for your character inventory, and still allows you to summon the totems without needed every individual one.  For some reason when i completed the quest, it didn't get put into my bag or my character inventory. I thought maybe it got sent to my mailbox, but it didn't go there either.  Without that relic, I am unable to summon any of my totems.
**Expected behaviour**: Tell us what should happen instead
After completing quest, you are suppose to receive a relic called Totem of the Earthen Ring.
**Steps to reproduce the problem**:

1. Collect all totems for shaman at level 30
2. Speak to Gotura Fourwinds to complete quest
3. Totems get destroyed from inventory and gets replaced by Totem of the Earthen Ring

http://www.wowhead.com/quest=14100/relic-of-the-earthen-ring#comments

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @dedraik.
I did this quest on PTR and I wasn't able to reproduce the issue.
![image](https://user-images.githubusercontent.com/6797611/31859718-140bc99a-b718-11e7-8355-d2a6f2662e9c.png)

Could you tell us exacly what did you do so item https://wowgaming.altervista.org/aowow/?item=46978 wasn't added to your inventory?

Note: For future reports please use https://wowgaming.altervista.org/aowow/ and add ids to quests, items, spells, npcs etc. Thank you.

**Comment by dedraik** [comment]
Hello and thank you for responding back to me so quickly. I greatly appreciate the help and efforts you put into resolving the issue.  Yes, unfortunately I don't have it in my bag or my equipped items.  I believe at the time my bags may have been full. It did let me complete the quest though and it removed my other totems from my bag.  I guess i might be able to see myself selling it, but i never deleted the item or anything.  Im not really sure what could have happened to it. If there is anything else i can supply or do I would like to help out anyway that I can. Thank you
 

    On Sunday, October 22, 2017 3:04 AM, Teridan <notifications@github.com> wrote:
 

 Hello and thank you for the report @dedraik.
I did this quest on PTR and I wasn't able to reproduce the issue.
Could you tell us exacly what did you do so item https://wowgaming.altervista.org/aowow/?item=46978 wasn't added to your inventory?Note: For future reports please use https://wowgaming.altervista.org/aowow/ and add ids to quests, items, spells, npcs etc. Thank you.—
